ros2node/ros2node/verb/info.py

```diff
@@ -49,8 +50,13 @@ def main(self, *, args):
                 publishers = get_publisher_info(node=node, remote_node_name=args.node_name)
                 print('  Publishers:')
                 print_names_and_types(publishers)
-                services = get_service_info(node=node, remote_node_name=args.node_name)
-                print('  Services:')
-                print_names_and_types(services)
+                service_servers = get_service_server_info(
+                    node=node, remote_node_name=args.node_name, include_hidden=args.include_hidden)
```

Comment:
On the `dashing` branch the `info` verb doesn't have the `--include-hidden` option. See #417.